app/main.py

```
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command == "":
+        return
+    command_list = command.strip().split()
+    if command_list[0] != "cp" or len(command_list) != 3:
+        return
+
+    try:
+        file_in = command_list[1]
+        file_out = command_list[2]
+        if file_in == file_out:
+            return
+
+        with (open(file_in, "r") as file_to_copy,
+              open(file_out, "w") as file_to_insert):
+            file_to_copy_content = file_to_copy.read()
+            file_to_insert.write(file_to_copy_content)
+    except FileNotFoundError:
+        print("FIle not found!!!")
```

This violates the 'Clean Code' requirement from the checklist. All debugging artifacts, including `print` statements, should be removed from the final submission. The function should probably fail silently by returning, which would be consistent with how other validation errors are handled.